mne_realtime/lsl_client.py

```diff
@@ -127,7 +127,7 @@ def _create_info(self):
             try:
                 montage = 'standard_1005'
                 info = create_info(ch_names, sfreq, ch_types, montage=montage)
-            except ValueError:
+            except TypeError:
```

Comment:
What you have currently will always hit the TypeError on latest MNE and never add a montage, because the montage argument is gone. A better fix here is to do something like:
```
info = create_info(ch_names, sfreq, ch_types)
try:
    info.set_montage('standard_1005', match_case=False)
except ValueError:
    pass
```